CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - [1330](https://github.com/umee-network/umee/pull/1330) Implemented MaxSupplyUtilization.
 - [1319](https://github.com/umee-network/umee/pull/1319) Implemented MaxSupply.
 - [1331](https://github.com/umee-network/umee/pull/1331) Implemented MinCollateralLiquidity.
+- [1343](https://github.com/umee-network/umee/pull/1343) RepayBadDebt and Liquidate automatically clear blacklisted collateral.
```

Comment:
This is the last required item on the v3 leverage checklist. It covers an edge case that would cause repeated inefficient no-ops every `EndBlock` if reserve exhaustion occurred after a collateral asset had to be blacklisted.